Plugin/Magento/Sales/Api/Data/OrderExtension.php

```diff
@@ -10,39 +10,51 @@
 
 use Magento\Framework\App\ObjectManager;
 
-class OrderExtension {
+class OrderExtension
+{
 
+    /**
+     * @var \Magento\Framework\HTTP\PhpEnvironment\Request
+     */
     protected $request;
 
-    public function __construct(\Magento\Framework\HTTP\PhpEnvironment\Request $request) {
+    /**
+     * @var ObjectManager
+     */
+    private $objectManager;
+
+    /**
+     * OrderExtension constructor.
+     *
+     * @param \Magento\Framework\HTTP\PhpEnvironment\Request $request
+     */
+    public function __construct(\Magento\Framework\HTTP\PhpEnvironment\Request $request)
+    {
        $this->request = $request->setPathInfo();
+       $this->objectManager = ObjectManager::getInstance();
     }
 
     /**
-     * Avoid default email is sent.
-     *
-     * With a MyParcel shipment, the mail should be sent only if the barcode exists.
+     * Use the delivery_options data from the sales_order table so it can be used in the magento rest api.
      *
-     * @return string|null
+     * @return mixed
      */
-    public function afterGetDeliveryOptions() {
-
-        $objectManager =  ObjectManager::getInstance();
-
-        $resource = $objectManager->get('Magento\Framework\App\ResourceConnection');
+    public function afterGetDeliveryOptions()
+    {
+        $resource = $this->objectManager->get('Magento\Framework\App\ResourceConnection');
         $connection = $resource->getConnection();
-        $tableName = $resource->getTableName('sales_order'); //gives table name with prefix
+        $tableName = $resource->getTableName('sales_order'); // Gives table name with prefix
 
-        $entity_id  = str_replace("/rest/V1/orders/","" ,$this->request->getPathInfo());;
+        $entityId  = str_replace("/rest/V1/orders/","" ,$this->request->getPathInfo());
 
         //Select Data from table
         $sql = $connection
             ->select('delivery_options')
             ->from($tableName)
-            ->where('entity_id = '. $entity_id);
+            ->where('entity_id = '. $entityId);
```

Comment:
->where('entity_id = '. $entityId);
            ->where('entity_id = ' . $entityId);
Dit moet je ook even in jouw phpstorm instellen

Reply:
Done!